README.md

```diff
@@ -35,7 +35,15 @@ there are two known configurations:
 
 This program defaults to the former. The latter requires commandline overrides:
```

Comment:
Maybe delete this paragraph if you're going to put in explicit commands for both?